README.md

```diff
@@ -2,4 +2,14 @@
 
 A quick demo of a bare vanilla JS todo app
 
-Also there to show the functionality of the [Edge Developer Tools for VS Code](https://aka.ms/devtools-for-code) extension.
\ No newline at end of file
+In `launch.json`, replace the "url" string by the path for `index.html`; for example, in three places, change:
+```
+"url": "c:\\Users\\myaccount\\.vscode\\extensions\\ms-edgedevtools.vscode-edge-devtools-1.4.7\\out\\startpage\\index.html", // Provide your project's url to finish configuring
+```
+to:
+
+```
+"url": "C:\\Users\\myaccount\\Documents\\GitHub\\simple-to-do\\index.html", // Provide your project's url to finish configuring
+```
+
```

Comment:
Readme should link directly to the docs as well as to the marketplace page that links to docs.  Heading toward a star linking design w/ DMC in the middle.